canonicalwebteam/discourse/parsers/docs_parser.py

```diff
@@ -516,6 +517,19 @@ def _replace_notifications(self, soup):
 
         return soup
 
+    def _add_anchor_support(self, soup):
```

Comment:
The function can be called `_add_anchors_headers`. 
`_add_anchors_support` sounds more like a commit message :)